tools/arch-visualizer/viewer/src/components/SearchOverlay.tsx

```
@@ -0,0 +1,196 @@
+                    ${i === selectedIndex
+                      ? darkMode
+                        ? "bg-blue-500/15 text-blue-300"
+                        : "bg-blue-50 text-blue-700"
+                      : darkMode
+                        ? "hover:bg-zinc-800/50"
+                        : "hover:bg-zinc-50"
+                    }
+                  `}
+                  onClick={() => handleSelect(result)}
+                  onMouseEnter={() => setSelectedIndex(i)}
+                >
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)